src/nyc_trees/apps/mail/templates/mail/online_training_complete.txt

```diff
@@ -0,0 +1,4 @@
+{% extends "mail/base_email.txt" %}
+{% block content %}
+Congratulations on completing the online training for TreesCount! 2015. Please sign up for an in-person training event at {{ events_url }} and help us reach our goal of mapping every street tree in New York City!
```

Comment:
Plain text templates are sensitive to whitespace. On other templates I have kept the block tags on the same line as the content to prevent inserting extra newlines into the output.